plugin/trino-cassandra/src/test/resources/cu-cassandra-latest.yaml

```diff
@@ -0,0 +1,1794 @@
+# Cassandra storage config YAML
```

Comment:
Could you put only required configuration likes cassandra-dse.yaml?

Reply:
Modified the file with the minimum configuration. Though I could not find any doc on the what are properties that are not required to start the Cassandra server.

With the current property set, Tests are passing.